manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -82,22 +82,47 @@ spec:
     rules:
     - alert: UsingDeprecatedAPIAppsV1Beta1
       annotations:
-        message: A client in the cluster is using deprecated apps/v1beta1 API that will be removed soon.
+        summary: "A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version"
```

Comment:
If this is meant for customers can we specify at least this is a "kubernetes client".